packages/components/src/components/password-input/password-input.jsx

```diff
@@ -31,6 +31,7 @@ const PasswordInput = ({
                         onClick={togglePasswordVisibility}
                         color='secondary'
                         width={18}
+                        data_testid='dc-password-input__visibility-icon'
```

Comment:
This needs to be in snake case as mentioned in the testing guideline document in WikiJs

Reply:
renamed the id